openstack/nova-hypervisor-agents/templates/etc/_nova.conf.tpl

```diff
@@ -0,0 +1,139 @@
+# nova.conf
+[DEFAULT]
+{{- range $k, $v := .Values.default }}
+{{ $k }} = {{ $v }}
+{{- end }}
+
+log_config_append = /etc/nova/logging.ini
+{{- include "ini_sections.logging_format" . }}
+state_path = /var/lib/nova
+
+# we patched this to be treated as force_resize_to_same_host for mitaka
+# https://github.com/sapcc/nova/commit/fd9508038351d027dcbf94282ba83caed5864a97
+allow_resize_to_same_host = true
+# but now we also need a new patch with this for queens
+always_resize_on_same_host = {{ .Values.always_resize_on_same_host | default false }}
+
+enable_new_services = {{ .Values.enable_new_services | default .Release.IsInstall }}
+service_down_time = {{ .Values.service_down_time | default 60 }}
+
+default_schedule_zone = {{.Values.global.default_availability_zone}}
+default_availability_zone = {{.Values.global.default_availability_zone}}
+
+rpc_response_timeout = {{ .Values.rpc_response_timeout | default .Values.global.rpc_response_timeout | default 60 }}
+
+sync_power_state_pool_size = {{ .Values.sync_power_state_pool_size | default 500 }}
+sync_power_state_interval = {{ .Values.sync_power_state_interval | default 1200 }}
+sync_power_state_unexpected_call_stop = false
+
+prepare_empty_host_for_spawning_interval = 600
+
+{{ template "utils.snippets.debug.eventlet_backdoor_ini" "nova" }}
+
+# usage refreshes on new reservations, 0 means disabled
+# number of seconds between subsequent usage refreshes
+max_age = {{ .Values.usage_max_age | default 0 }}
+# count of reservations until usage is refreshed
+until_refresh = {{ .Values.usage_until_refresh | default 0 }}
+
+{{- include "osprofiler" . }}
+
+{{ include "ini_sections.oslo_messaging_rabbit" .}}
+
+[oslo_concurrency]
+lock_path = /var/lib/nova/tmp
+
+[glance]
+num_retries = 10
+
+[cinder]
+os_region_name = {{.Values.global.region}}
+cross_az_attach = {{.Values.cross_az_attach}}
+auth_url = {{ .Values.identity_service_url }}
+auth_type = v3password
+username = {{ .Values.global.nova_service_user | default "nova" }}
+password = {{ required ".Values.global.nova_service_password is missing" .Values.global.nova_service_password }}
```

Comment:
While you did add the secrets via extra config file, they're also still in this file which ends up in a ConfigMap.